raiden_contracts/tests/test_channel_open.py

```diff
@@ -55,16 +56,81 @@ def test_max_1_channel(token_network, get_accounts, create_channel):
         token_network.functions.openChannel(B, A, TEST_SETTLE_TIMEOUT_MIN).transact()
 
 
+def test_participants_hash(token_network, get_accounts):
+    (A, B) = get_accounts(2)
+
+    AB_hash = get_participants_hash(A, B)
+    assert token_network.functions.getParticipantsHash(A, B).call() == AB_hash
+
+
+def test_counter(token_network, get_accounts, create_channel):
```

Comment:
do we have a test for the replay? I think this is the more important thing that this PR fixes, the nonce is just a implementation detail

Reply:
Added replay tests for all the signed messages that we have - for reopened channels.